spec/controllers/giraffes_controller_spec.rb

```diff
@@ -36,10 +36,12 @@
       expect(response.status).to eq 200
       expect(response.content_type).to eq("application/json")
 
-      expect(returned_json.length).to eq 3
+      expect(returned_json.length).to eq 4
       expect(returned_json["name"]).to eq first_giraffe.name
       expect(returned_json["id"]).to eq first_giraffe.id
       expect(returned_json["description"]).to eq first_giraffe.description
+      
+      expect(returned_json["reviews"]).to eq first_giraffe.reviews
```

Comment:
Lets make sure to add some reviews to `first_griaffe` above, so that we can ensure they are returning correctly